docs/data_usage_guides/metadata.md

```diff
@@ -0,0 +1,166 @@
+[`Food Security`](food_security_and_nutrition.md#food-security),
+[`Food Prices`](food_security_and_nutrition.md#food-prices),
+[`Baseline Population`](population_and_socio-economy.md#population)
+
+## Sub-category Metadata
+
+### Org <a id="org"></a>
+
+**Used in:**
+[`3W - Who is Doing What Where`](coordination_and_context.md#operational-presence)
+
+The organisation table is populated from the 3W data, using the following
+methodology:
+
+* Organisation name and acronym strings are normalised. If an acronym isn’t
+  available, the first 32 characters of the name are used.
+* This [organisation mapping](https://docs.google.com/spreadsheets/d/e/2PACX-1vSfBWvSu3fKA743VvHtgf-pIGkYH7zhy-NP7DZgEV9_a6YU7vtCeWhbLM56aUL1iIfrfv5UBvvjVt7B/pub?gid=1040329566&single=true&output=csv)
+  is used for common alternative names
+* Organisations must have an associated organisation type. If available, the
+  organisation type code is taken directly from the 3W data, otherwise the name
+  string is normalised and matched to the org type names. In the absence of a
+  direct match, phonetic matching is used for strings > 5 characters. If no
+  match is found, the organisation is skipped.
+
+### Org Type <a id="org-type"></a>
+
+**Used in:**
+[`Org`](org),
+[`3W - Who is Doing What Where`](coordination_and_context.md#operational-presence)
+
+The table is populated using
+[OCHA Digital Services organization types list](https://data.humdata.org/dataset/organization-types-beta),
+with the addition of:
+
+* Civil Society
+* Observer
+* Development Programme
+* Local NGO.
+
+Organisation types all have an associated name and code.
+
+### Sector <a id=sector></a>
+
+**Used in:**
+[`3W - Who is Doing What Where`](coordination_and_context.md#operational-presence),
+[`Humanitarian Needs`](affected_people.md#humanitarian-needs)
+
+This table is populated using the Global Coordination Groups, with the
+following additional entries:
+
+* cash
+* humanitarian assistance
+* multi-sector
+* intersectoral.
+
+The sector name strings in the 3W data are normalised and then aligned to the
+sector table, using the “sector_map” section of
+[this configuration file](https://github.com/OCHA-DAP/hapi-pipelines/blob/main/src/hapi/pipelines/configs/core.yaml)
+if needed.
+In the absence of a direct match, phonetic matching is used for
+strings > 5 characters.
+
+### Currency <a id="currency"></a>
+
+**Used in:**
+[`Food Prices`](food_security_and_nutrition.md#food-prices)
+
+The currency table is populated using the WFP VAM Data Bridges API.
+
+### WFP Market <a id="wfp-market"></a>
+
+**Used in:**
+[`Food Prices`](food_security_and_nutrition.md#food-prices)
+
+Markets are defined as the physical locations where buyers and sellers
+come together to trade goods and services. This table is populated from the
+food prices data, which contains location names down to admin 2 as well as
+a latitude and longitude, however is not p-coded.
+We therefore p-code .... TODO
```

Comment:
This table is populated from the food prices data, which contains location names down to admin 2 as well as
a latitude and longitude, but is not p-coded. Consequently, admin 2 names must be matched to p-codes using the algorithm provided by the hdx-python-country library which includes phonetic name matching and manual overrides.